README.md

```diff
@@ -1,5 +1,19 @@
-# armory_examples
+# Armory Example Projects
 
-Examples to accompany Armory SDK. Check out the [manual](https://github.com/armory3d/armory/wiki).
+Browse online: https://armory3d.github.io/armory_examples_browser/
 
-Note: Download the package from [releases](https://github.com/armory3d/armory_examples/releases). Only use the latest revision of this repository if you are using [git Armory](https://github.com/armory3d/armory/wiki/gitversion).
+Only use the latest revision of this repository if you are using [armory-git](https://github.com/armory3d/armory/wiki/gitversion).
```

Comment:
Is it safe to remove link to releases for newcomers?

Reply:
I think it is, since the releases aren't updated frequently, nor are there any checks if the 'releases' are more stable. So the current head is probably more stable than the latest release (2022.6).

Reply:
👍